tests/integration/tasks/test_databricks.py

```diff
@@ -230,3 +230,41 @@ def test_discover_schema():
     }
 
     assert schema_result == expected_schema
+
+
+def test_rollback():
+    append_data = [
+        {
+            "Id": "UpsertTest2",
+            "AccountId": 789,
+            "Name": "new upsert-2",
+            "FirstName": "Updated",
+            "LastName": "Carter2",
+            "ContactEMail": "Adam.Carter@TurnerBlack.com",
+            "MailingCity": "Updated!Jamesport",
+        }
+    ]
+
+    table = "test_rollback"
+    fqn = f"{schema}.{table}"
```

Comment:
There is a convention to use CAPS for constants in Python. In this case, `SCHEMA` is a constant as you're using the same schema for all tests.